vector/src/main/res/values/strings.xml

```diff
@@ -2164,15 +2164,13 @@
     <string name="send_suggestion">Make a suggestion</string>
     <string name="send_suggestion_content">Please write your suggestion below.</string>
     <string name="send_suggestion_report_placeholder">Describe your suggestion here</string>
-    <string name="send_suggestion_sent">Thanks, the suggestion has been successfully sent</string>
-    <string name="send_suggestion_failed">The suggestion failed to be sent (%s)</string>
```

Comment:
lint won't like that, you have to also delete all the translations. Maybe just keep it for now and we will do the cleanup later

Reply:
done